### ISSUE TITLE
2024_feb_epic: Copy  

### ISSUE THREAD

**Issue by ljmoody** [lzim/teampsd#3078]
- [ ] Copy the steps and screenshots from #743 into Team PSD manual, Chapter X - "Steps for Refreshing the Data Model for each VISN Report using A01 Server Connection for PTSD_OMHO"
- [ ] potential title for new section: Refresh the Data in the MTL Data UI
- [ ] check against [Lucidchart](https://lucid.app/lucidchart/0cd93668-a02d-4176-b97d-4f342985a57f/edit?invitationId=inv_3462b1a9-3207-4813-87e1-993924811a3f&page=0_0#) and existing manual documentation:
  - 7.15: Schedule Daily Data UI Refresh
  - 7.9 Refresh Data for R01 or IIR

**Comment by lzim** [comment]
@jamesmrollins  & @ljmoody _Is this done?_

**Comment by ljmoody** [comment]
@lzim & @jamesmrollins, 
From what I can tell, the information was updated - and I'm unable to find more updated step-by-step instructions. Furthermore,

1. I don't see the relevance in card [#743](https://github.com/lzim/teampsd/issues/743) to this card title and task. And realize the irony here since I created the card and mentioned that issue #. 🤦 
2. The current Team PSD Manual includes captured information found in [Lucidchart](https://lucid.app/lucidchart/0cd93668-a02d-4176-b97d-4f342985a57f/edit?invitationId=inv_3462b1a9-3207-4813-87e1-993924811a3f&page=0_0#). 
    - Sections 7.10 (Refresh Data for R01 or IIR) and 7.16 (Schedule Daily Data UI Refresh) were updated 9 months ago with this information.
    - This note indicates that I couldn't find one video mentioned, and did not have (and haven't been given) access to the 9/20/22 video: 
![MicrosoftTeams-image (2)](https://github.com/lzim/teampsd/assets/124643236/666c81f6-665b-4477-9764-73110ed2c91b)
3. I found this [Sept 2022 Teams post](https://teams.microsoft.com/l/message/19:d15133fbfb4d4c3a8c81701292b1890d@thread.skype/1662998586880?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1662998586880&teamName=teampsd_vha&channelName=training_workflow&createdTime=1662998586880) re: Refresh Data for R01 or IIR.

IF it is possible to get links to either or both of the videos mentioned in point 2 above, I can add them in.
If not, I suggest closing this card as complete since all documentation I can find is captured.

**Comment by lzim** [comment]
@ljmoody I'm wondering if the link between issue cards is to [research/743](https://github.com/lzim/research/issues/743) not teampse/743

I think that this is a question for **#quant_workflow**
 @dkngenda 
 @kmacia
 @lijenn 
 @tanyapod 
 
 _Can Laura close this card? Does Chapter 7.9 of the Team PSD manual at https://mtl.how/teampsd_manual describe the training conducted in March for data refreshes sufficiently?

**Comment by ljmoody** [comment]
@lzim, that was the missing link (literally)! Yes, that research card makes sense. In which case, no - the existing manual does not have the information or screen captures contained in [Research 743](https://github.com/lzim/research/issues/743). 

_Do the numbered steps and screenshots from your Feb 5 comment on that card contain all information needed to update the data refresh section within the PSD Manual?_

**Comment by kmacia** [comment]
I don't know the answer to this question (I'm not familiar with the content on [research/743](https://github.com/lzim/research/issues/743) in general), but maybe someone else on #quant_workflow knows the answer.

**Comment by lzim** [comment]
@ljmoody - Let's add this to our agenda for **8am_workflow** 

- The initial requirements list on this card covers a couple things:

01. **https://mtl..how/data** - How to update the Data Model for the _MTL Data User-Interface_ Jeff Hoerle documented this here: https://github.com/lzim/research/tree/master/data_ui and this documentation is completed.

02. **https://mtl.how/teampsd_dashboard** - How to update our Data Feeds for tests of specific aims in our trials @dkngenda is working on this, but I'm not sure the state of that documentation. _David?_

03. **https://mtl.how/research** - How to update the R Notebook for each grant using GitHub was covered in the March training #quant_workflow had hosted by Dr. Ngendahimana. I scribed this starting [here](https://teams.microsoft.com/l/message/19:09778e4391404920a8a20c7ef4ec3de3@thread.skype/1710880939609?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1614716378260&teamName=teampsd_vha&channelName=all_teampsd&createdTime=1710880939609) and Jenn continued it the following week.

**Comment by lzim** [comment]
@matomasz & @ljmoody 
_Should this be added to **#9am_workflow_leads** or **#support_workflow** for **wk3 2024_07**?_

**Comment by lijenn** [comment]
Needs to be discussed at wk1 2024_08 meetings to disposition.

**Comment by lijenn** [comment]
Discussed sim_ui_workflow 8/7:
@lijenn @jamesmrollins @ljmoody @matthewtomo Triaged to 2024_10 given current 3.7 bugs and MTL documentation priorities.

**Comment by lzim** [comment]
@ljmoody - _Do you know what this is?_

I'm not sure I'm tracking what this is from the card.
Thanks!

FYI: @matthewtomo

**Comment by lzim** [comment]
@ljmoody 
I agree - _Are you willing to create that card?_
> @lzim, regarding point 3 below, I think it makes sense for a new card to be created to track that documentation addition to the Team PSD manual in the future. Do you agree?

**Comment by ljmoody** [comment]
@lzim, a quick update on the status of this card:

- [ ] 1. https://mtl.how/data - documentation is complete per your comment above. _Is the linked research repo's readme sufficient or does this documentation also need to be added to the Team PSD Manual?_ 
- [ ] 2. https://mtl.how/teampsd_dashboard - waiting on input from @dkngenda 
- [ ] 3. https://mtl.how/research - created card #3286 to track this task & effort

**Comment by dkngenda** [comment]
@lzim @hnorthcraft @lijenn 

Need clarification on what my ask on this card is. Thank you

**Comment by lzim** [comment]
No problem @dkngenda - Thanks for reviewing these.

I believe the issue here is that we don't have documentation in the Team PSD Manual at https://mtl.how/teampsd_manual about how to update the https://mtl.how/teampsd_dashboard using an .Rmd notebook on GitHub. 

We do have https://mtl.how/research/power_bi

**Comment by dkngenda** [comment]
@lzim I am assuming the second bullet point above is directed at another person? Let me know.

Thank you
